js/index.js

```diff
@@ -198,6 +204,27 @@ Server.prototype = {
       }
     };
     return this.webApp.installPlugin(pluginContext);
+  },
+
+  checkProxiedHost: function (host, port) {
```

Comment:
The code looks alright but I would discourage adding stuff to index.js directly. The purpose of index.js is to do the wiring between components, and not to be a dump  for "miscellaneous" functionality, even if it looks small.

Please move this to, let's say, proxy.js

Reply:
That works for me too.